### DISCUSSION TITLE
upscaling on laptops

### DISCUSSION THREAD

**Discussion by ConteXCrown** [bloc97/Anime4K#176] [Q&A] [answered]
My Laptop has a i7 1065g7 and when I use the upscaler the gpu usage is always at 100%.

So what happens when my gpu is too slow to use the upscaler, will there be more latency or not the whole image is being processed?

thank you for response btw this project is awesome
![gpu 100](https://user-images.githubusercontent.com/98663891/170776955-5d424f82-4c33-4818-b8cd-86dffd99b348.PNG)

**Comment by bloc97** [comment, marked as answer]
The default behavior of mpv when the GPU can't keep up is to drop (skip) one or many frames, until the GPU catches up.
If that's the case you will see stuttering. For integrated graphics you might need to reduce the settings even further, as the two configurations are optimized for desktop GPUs.
